app/controllers/api/submissions_controller.rb

```diff
@@ -24,6 +24,10 @@ def update
     end
 
     def index
+      # Return x-total-count header if total_count param is present
+      if params[:total_count].present?
+        headers["X-Total-Count"] = Submission.count
```

Comment:
**suggestion**: This would always return the total number of Submissions in the database. We should push this further down below any scopes/filters, if we need it. What UI would this support?

Reply:
> This would always return the total number of Submissions in the database. We should push this further down below any scopes/filters

Good catch, will update

> What UI would this support?

I added this to keep the API the same as we do with Gravity for connections
<img width="683" alt="Screenshot 2024-06-25 at 14 00 55" src="https://github.com/artsy/convection/assets/11945712/e7303c77-3a56-4461-ac37-5e7a2589332d">